utils/merge_and_push_translations.sh

```diff
@@ -96,30 +96,16 @@ fi
 python3 utils/fix_formatting.py --lua --dir data/i18n
 python3 utils/fix_formatting.py --lua --dir data/txts
 
-# Undo one-liner diffs in po directory - these are pure timestamps with no other content
-if false; then  # Currently disabled due to TX no longer updating timestamps reliably
+# Undo one-liner diffs of pure timestamps with no other content
 set +x
-nrAdded=""
-nrDeleted=""
-for entry in $(git diff --numstat po); do
-  if [ -z "$nrAdded" ]
+for entry in $(git diff --numstat po/*/*.pot | sed -En '/^1\t1\t/s/^1\t1\t(.*)/\1/p'); do
```

Comment:
I don't really like this `sed` command, but I don't know if it could be simplified. Other alternative would be `grep` and `cut`, but `sed` at least does it all at once.

Reply:
OK, didn't know `p` at the end is not another command, but a flag of `s`, and it only prints if there was a substitution. Now I do. :)